specification/frontdoor/resource-manager/readme.md

```diff
@@ -49,6 +49,24 @@ directive:
 ```
 
 
+### Tag: package-2020-04
+
+These settings apply only when `--tag=package-2020-04` is specified on the command line.
+
+``` yaml $(tag) == 'package-2020-04'
+input-file:
+- Microsoft.Network/stable/2020-01-01/network.json
+- Microsoft.Network/stable/2019-11-01/networkexperiment.json
+- Microsoft.Network/stable/2020-04-01/frontdoor.json
+- Microsoft.Network/stable/2020-04-01/webapplicationfirewall.json
```

Comment:
Please update `package-2020-05` above with the new version

Reply:
I think I already did... Did I miss something else ?

Reply:
This line in the same readme file: https://github.com/Azure/azure-rest-api-specs/pull/9383/files/00f8c8a3ea5dbaffcb1bccaa768f77024628d3d7#diff-02bd54d5afdfb095a174499ad32a314bR43
```

``` yaml $(tag) == 'package-2020-05'
input-file:
- Microsoft.Network/stable/2020-05-01/network.json
- Microsoft.Network/stable/2019-11-01/networkexperiment.json
- Microsoft.Network/stable/2020-05-01/frontdoor.json
- Microsoft.Network/stable/2019-10-01/webapplicationfirewall.json
```